gulpfile.js

```diff
@@ -31,10 +54,21 @@ gulp.task('test:jshint', function() {
 
 // Make the Mocha task depend on the JSHint task. Otherwise, the tasks will be
 // run in parallel and will print to console simultaneously.
-gulp.task('test:mocha', ['test:jshint'], function() {
-    gulp.src('tests/tests.js')
-        .pipe(mocha())
-        .once('end', function() {
-            process.exit();
-        });
+gulp.task('test:mocha', ['build', 'test:jshint'], function() {
+    var mocha = require('gulp-mocha');
+
+    process.env.NODE_ENV = 'test';
+
+    // Create a fresh database
```

Comment:
`sequelize.sync({ force: true })` creates fresh tables, but not a fresh database. I wanted to create the database programmatically so that the person running the tests has one less manual step.